### ISSUE TITLE
Cannot query ACF Options with GraphQL

### ISSUE THREAD

**Issue by k3lh4m** [gatsbyjs/gatsby#9615]
I am trying to query my ACF options page fields that I have set up in the admin side, and through the WP API Query them using GatsbyJS's gatsby-source-wordpress. When I run build and dev it requests the data and prints out the output

```
=== [ Fetching wordpress__acf_options ] === http://url.local/wp-json/acf/v3/options/options/
⠈ source and transform nodes -> wordpress__acf_options fetched : 1
Fetching the wordpress__acf_options took: 518.885ms

=== [ Fetching wordpress__acf_options ] === http://url.local/wp-json/acf/v3/options/options
⠈ source and transform nodes -> wordpress__acf_options fetched : 1
Fetching the wordpress__acf_options took: 478.580ms
```

and within those urls on the browser, I get this response. So I know the data is correct

```
 {
        acf: {
            social_options: [
                {
                     social_name: "Twitter ",
                     social_link: "https://twitter.com"
                },
                {
                     social_name: "Linkedin",
                     social_link: "https://www.linkedin.com/"
                }
            ],
            test: "pagfe 1"
        }
    }
```

The problem is when I query the allWordpressAcfOptions in the internal GraphQL. I try and see what options are available to query but all I can get is the ID. This is my query below.

```
{
  allWordpressAcfOptions {
    edges {
      node {
        id
        internal {
          type
          contentDigest
          owner
        }
        children {
          id
        }
        wordpress_id
      }
    } 
  }
}
```

and none of the data in there is the data I need, an none of it is representative of the same data shown from REST API Link. So there is something missing. This is the response I get back from the Query.

```
{
  "data": {
    "allWordpressAcfOptions": {
      "edges": [
        {
          "node": {
            "id": "038ece9f-326d-5bb0-8f17-1bb72ce06c57",
            "internal": {
              "type": "wordpress__acf_options",
              "contentDigest": "10b2729a5f87723bbafc539f99fb23d6",
              "owner": "gatsby-source-wordpress"
            },
            "children": [],
            "wordpress_id": "acf"
          }
        }
      ]
    }
  }
}
```
For reference my gatsby-source-wordpress options in the gatsby config are this below

```
options: {
    baseUrl: 'url.local',
    protocol: 'http',
    hostingWPCOM: false,
    useACF: true,
    acfOptionPageIds: ['options'],
    verboseOutput: true
},
```
options refers to the page ID the my ACF options page is set in my functions.php on the wordpress side.

So in summary, GatsbyJs is referencing the API and pulling the data but what am I doing wrong to query the data via GraphQL?

**Comment by k3lh4m** [comment]
Anybody able to take a look at this?

**Comment by yumyo** [comment]
Did you ever solve this @k3lh4m ?

**Comment by lnikell** [comment]
For me the issue was because of using 2 plugins for ACF API. 
I had installed ACF to WP API and ACF to Rest API. Because of this line in the gatsby wordpress plugin 
https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-wordpress/src/fetch.js#L456
it defines acf api version based on the first match in found routes. Better to fix it by allowing users to set api version through the config.

**Comment by lexcode** [comment]
I am facing the same issue here =/

Plugins installed 
![image](https://user-images.githubusercontent.com/349424/61474557-d1117100-a980-11e9-9271-516e1a09f701.png)


Can't find ACF here 
![image](https://user-images.githubusercontent.com/349424/61474597-e7b7c800-a980-11e9-83fa-50aa1ce69235.png)

**Comment by diegoavalon** [comment]
The above setup looks correct to me. Have you set your request API version to 3? Found in the admin dashboard: Settings > Permalinks

![Screen Shot 2019-09-13 at 1 47 31 PM](https://user-images.githubusercontent.com/8173272/64894438-9e0cf600-d62e-11e9-9d00-208d73545ec0.png)

**Comment by DavidTheSimon** [comment]
I am facing the same issue. Any ideas are appreciated.

**Comment by rwschmitz** [comment]
@lexcode @DavidTheSimon Are you guys providing the necessary subfields for your ACF fields?  You'll have to do that if you're not.

My GraphQL query:

<img width="836" alt="Screen Shot 2019-09-26 at 7 47 27 PM" src="https://user-images.githubusercontent.com/30060083/65733989-8ace4180-e096-11e9-83cf-1e139427c8b3.png">

My ACF setup:

<img width="788" alt="Screen Shot 2019-09-26 at 7 48 24 PM" src="https://user-images.githubusercontent.com/30060083/65734015-a46f8900-e096-11e9-8967-58da6d42974e.png">

My plugins:

<img width="1098" alt="Screen Shot 2019-09-26 at 7 48 59 PM" src="https://user-images.githubusercontent.com/30060083/65734031-b2250e80-e096-11e9-957e-2759cad66f04.png">
